docs/source/app_development/create.rst

```diff
@@ -204,4 +204,11 @@ Typically one would prefer to use the cookie-cutter to create a new app, but her
 
    In case you encounter any issues, compare your app to the source code of the `hello world app <https://github.com/aiidalab/aiidalab-hello-world>`_ that was templated by the cookiecutter approach.
 
+  .. note::
+
+       The UID and GID of folder bind to container may different from the user's UID and GID in host machine,
+       Which will lead to edit files in bind mount aiidalab folder permission denied.  
+       If you are using vscode as IDE, there is a `Attach to a running container <https://code.visualstudio.com/docs/remote/attach-container>` 
```

Comment:
Please fix the hyperlink, this is [restructured text](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html), not markdown.